PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/data/dao/PlantDao.java

```diff
@@ -0,0 +1,24 @@
+package edu.cascadia.mobas.photopoints.data.dao;
+
+import androidx.room.Dao;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.Query;
+import androidx.room.Update;
+import java.util.List;
+import edu.cascadia.mobas.photopoints.data.dto.DBPlant;
+
+@Dao
+public interface PlantDao {
+    @Query("SELECT PlantID, Description, Species FROM Plants")
+    List<DBPlant> getPlants();
+
+    @Insert
+    void insert(DBPlant user);
+
+    @Delete
+    void delete(DBPlant user);
+
+    @Update
+    void update(DBPlant user);
```

Comment:
`insert`, `delete`, and `update` should take an argument named `plant` instead of `user`